example/app/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth import views as auth_views
+try:
+    from django.urls import include, url
```

Comment:
Hmm, guessing this is Django 1.11 or older? Does this project need to work with 2.x?

Reply:
`oidc-provider` looks it support 1.11.x, 2.x+. This is actually a `try..except` to work with both. The `tox` tests cover both as well and they all pass so it should be good.